### PR TITLE
Test sampling

### PR DESCRIPTION
**Pull request by laurentheirendt**
CI integration of `sampling/` tests

**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the guidelines in the [contributing](https://github.com/opencobra/cobratoolbox/blob/master/.github/CONTRIBUTING.md) document
- [X] Followed the [code styleguide](https://github.com/opencobra/cobratoolbox/blob/master/.github/CONTRIBUTING.md#code-styleguide)
- [X] Checked to ensure there aren't other open [Pull Requests](https://github.com/opencobra/cobratoolbox/pulls) for the same update/change
- [X] Written platform-independent code
- [X] Written a sensible test according to the [Test styleguide](https://github.com/opencobra/cobratoolbox/blob/master/.github/CONTRIBUTING.md#test-styleguide)
- [X] Tested your PR locally prior to submission
- [X] Checked that all tests pass
- [X] Tested the code on Linux

### CODE REVIEW

File: test/verifiedTests/testSampling/testGpSampler.m [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        for i = 1:length(samplePoints)
+            % call sampler
+            [sampleStructOut, mixedFrac] = gpSampler(model, samplePoints(i), [], 2);
+
+            % check
+            [errorsA, errorsLUB, stuckPoints] = verifyPoints(sampleStructOut);
+
+            tmpErrorsA = ~any(errorsA);
+
+            assert(tmpErrorsA(1) && tmpErrorsA(2))
```
Comment (syarra) [inline test/verifiedTests/testSampling/testGpSampler.m]: Could we use `all(tmpErrosA)`?

Comment (laurentheirendt) [inline test/verifiedTests/testSampling/testGpSampler.m]: I tried that, but `assert()` does like `all()`...

Comment (syarra) [inline test/verifiedTests/testSampling/testGpSampler.m]: does like?

Comment (syarra) [inline test/verifiedTests/testSampling/testGpSampler.m]: `testOptKnock` makes use of it and it seems to work.

**Pull request merged**